massgov/mayflower
# DP-11102  responsive table headers obscured by sticky menus

Responsive table header is obscured by sticky TOC
Responsive tables obscure the last row of the table

## Related Issue / Ticket

- [Responsive table header is obscured by sticky TOC DP-11102](https://jira.mass.gov/browse/DP-11102)
- [Responsive tables obscure the last row of the table DP-11026](https://jira.mass.gov/browse/DP-11026)

## Steps to Test


1. Visit https://mayflower.digital.mass.gov/b/DP-11102--responsive-table-headers-obsured-by-sticky-menus/patterns/05-pages-information-details-toc-and-responsive-tables/05-pages-information-details-toc-and-responsive-tables.html to see a page with a responsive table and sticky TOC

2. Scroll the table up the screen to see it stick below the stick toc

3. Scroll the table up to see the last row is no longer obscured by the stick horizontal nav

## Screenshots
![stickytoc](https://user-images.githubusercontent.com/18662734/49032116-67ceba80-f171-11e8-8a4c-4c341aa24605.png)


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
